tests/__init__.py

```diff
@@ -8,3 +10,15 @@
 def read_file_content(file_name):
     with open(file_name, 'rb') as open_file:
         return open_file.read()
+
+
+def create_crossref_config(config_section='elife'):
+    """utility to create the crossref object"""
+    raw_config_object = raw_config(config_section)
+    return parse_raw_config(raw_config_object)
+
+
+def create_article_object(doi='10.7554/eLife.00666', title='Test article'):
+    """create a basic article object"""
+    article = Article(doi, title)
```

Comment:
is this all it will ever be? Repetition of code in tests isn't a bad thing. Tests should be dumb, blunt and forceful. They precisely describe how the world *should* be. Keep an eye on functions like this that exist only to pretty-up tests, they slowly accumulate custom logic until eventually your tests need tests themselves.

Reply:
It looked like some repetitive code that could be reduced since every test case was using the same values. However, I can reduce the number of lines updated in this PR, and keep it more on-topic if I revert all these back to the way they were. Either way, I'm happy.

Reply:
I want to apologise if I'm being condescending. This is how I talk to myself when I do my own code reviews.

Removing repetition in regular code is a good impulse, but I think it's less suited while testing.

Reply:
> Tests should be dumb, blunt and forceful. They precisely describe how the world should be

I enjoyed this bit, whether the world is dumb, blunt and forceful now, or whether the world should be that way, is an interesting aspect to consider.

No worries Luke, I did not find any comments as condescending, it is your opinion expressed. Everyone's style is different, and the feedback and changes eventually arrives at a better result in the end.